web3deployment/KatalystDeploy/katalystNetworkDeployer.js

```diff
@@ -236,9 +249,8 @@ async function main() {
 
   chainId = chainId || await web3.eth.net.getId()
   console.log('chainId', chainId);
-  console.log('starting compilation');
-  output = await require("../compileContracts.js").compileContracts("sol5");
-  console.log("finished compilation");
+  console.log('retrieving artifacts...');
+  output = await require("../retrieveArtifacts.js").retrieveArtifacts();
```

Comment:
this means have to compile before starting?

what if compiled version is wrong?
or not the expected one?

do handle this situation?